### DISCUSSION TITLE
Prompting for specific context.

### DISCUSSION THREAD

**Discussion by bchkv** [absadiki/subsai#84] [Q&A] [answered]
Is there a way to prompt the utility with words for a certain context when it struggles to pick narrowly-used terms (e.g. Japanese origin words used in martial arts videos)?

**Comment by absadiki** [comment, marked as answer]
@bchkv, I think the `prompt` parameter can be used for this type of use cases. 
If you are using `openai/whisper` for example, you can provide the prompt through the [`prompt` config](https://abdeladim-s.github.io/subsai/#subsai.models.whisper_model.WhisperModel.config_schema).
Other models can be used the same way if they provide a `prompt` config, you can check the docs to know the available configs for every model.
